main.c

```diff
@@ -787,11 +790,9 @@ int main(int argc, char **argv)
 	struct tcmulib_context *tcmulib_context;
 	darray(struct tcmulib_handler) handlers = darray_new();
 	struct tcmur_handler **tmp_r_handler;
-	struct tcmu_config *cfg;
 
-	cfg = tcmu_config_new();
-	tcmu_load_config(cfg, NULL);
-	tcmu_set_log_level(cfg->log_level);
+	tcmu_cfg = tcmu_config_new();
+	tcmu_load_config(tcmu_cfg, NULL);
 
```

Comment:
Just a heads up. I merged PR

https://github.com/open-iscsi/tcmu-runner/pull/181

which adds a null check so it will break this part of the patch.